mock-services/DAG-parser.py

```diff
@@ -0,0 +1,337 @@
+import json
+import sys
+import yaml
+import os
+import shutil
+import stat
+
+class TerraformTemplate:
+    def __init__(self, services):
+        string = ""
+        for i, s in enumerate(services):
+            if i != 0:
+                string += " && "
+            string += f"kubectl create configmap {s}-configmap --from-file {s}/config.yaml"
+
+        self.template = \
+f"""service_account_key_path = "/Users/gordonwu/Downloads/ordinal-motif-270122-b60019637135.json"
+password = "TqzreFzPqXWKdDWQVfsj6Uv3ZX"
+bookinfo_apps_path = "test.yaml"
+add_configmap = "{string}"
```

Comment:
This is the only part of `TerraformTemplate` that's changing. The rest are all sensitive values or wouldn't work on my machine. 

Can you place make it so that `TerraformTemplate` only returns this string, and if we need to add it to `values.tfvars`, we can append if it does not exist?